tests/phpunit/integration/Core/Tags/Guards/Tag_Production_GuardTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Tag_Production_GuardTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Tags\Guards
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Tags\Guards {
```

Comment:
Let's remove curly brackets since we don't use them anywhere.
```suggestion
namespace Google\Site_Kit\Tests\Core\Tags\Guards;
```

Reply:
https://github.com/google/site-kit-wp/pull/3734/commits/5d02b7481e6870acc3b4ba472ce463e116432196